bin/test

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+playbooks=$(ls *.yml)
```

Comment:
with @jmaupetit we agree that playbooks should be moved to a `playbooks` repository. There are now too many and we have more to add...

Reply:
Can we do this on the issue #32 ?

Reply:
I think it should be addressed in a dedicated PR to ease its review (see #9)